docs/src/datasets/FashionMNIST.md

```diff
@@ -56,8 +56,8 @@ the Fashion-MNIST dataset in Julia more convenient.
 
 Function | Description
 ---------|-------------
-[`convert2features(array)`](@ref FashionMNIST.convert2features) | Convert the Fashion-MNIST tensor to a flat feature matrix
-[`convert2image(array)`](@ref FashionMNIST.convert2image) | Convert the Fashion-MNIST tensor/matrix to a colorant array
+[`convert2features(array)`](@ref MNIST.convert2features) | Convert the Fashion-MNIST tensor to a flat feature matrix
```

Comment:
does the `FashionMNIST.`one not work anymore? The idea is that even though it literally is just the MNIST one, it could always be replaced with its own implementation as well as everything needed for FashionMNIST is available in its module

Reply:
`@ref FashionMNIST.` does work but the hyperlink redirects to the same html page as `@ref MNIST.`, which I found confusing. So I wanted to make it explicit by switching to `@ref MNIST.` and adding a comment below saying that those two functions are just a re-export from MNIST. I can switch back to `@ref FashionMNIST.` if you think that would be more future proof, I'm indifferent to either choice.